pom.xml

```diff
@@ -1570,48 +1570,24 @@
                 <groupId>org.wildfly.wildfly-http-client</groupId>
                 <artifactId>wildfly-http-ejb-client</artifactId>
                 <version>${version.org.wildfly.http-client}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>*</groupId>
-                        <artifactId>*</artifactId>
-                    </exclusion>
-                </exclusions>
```

Comment:
If this is not removed then transitive dependencies are not picked up correctly when versions are managed from the EJB client bom.